cruise-control-metrics-reporter/src/test/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporterTest.java

```diff
@@ -202,4 +202,15 @@ public void testUpdatingMetricsTopicConfig() throws ExecutionException, Interrup
     topicDescription = adminClient.describeTopics(Collections.singleton(TOPIC)).values().get(TOPIC).get();
     assertEquals(2, topicDescription.partitions().size());
   }
+  @Test
+  public void testGetKafkaBootstrapServersConfigure() throws ExecutionException, InterruptedException {
+    String bootstrapServers = "";
+    for (Map<Object, Object> brokerConfig : buildBrokerConfigs()) {
+      if (brokerConfig.containsKey("listeners")) {
+         String listeners = String.valueOf(brokerConfig.get("listeners"));
```

Comment:
Can we refer to `KafkaConfig.ListenersProp()` instead of hardcoding `"listeners"`?